pegasus/test/test_v2_section_routes.rb

```diff
@@ -0,0 +1,191 @@
+# Tests for the routes in v2_section_routes.rb
+
+require 'minitest/autorun'
+require 'rack/test'
+require 'mocha/mini_test'
+require_relative 'fixtures/fake_dashboard'
+require_relative 'fixtures/mock_pegasus'
+
+class V2SectionRoutesTest < Minitest::Test
+  describe 'Section Routes' do
+    before do
+      FakeDashboard.use_fake_database
+      $log.level = Logger::ERROR # Pegasus spams debug logging otherwise
+      @pegasus = Rack::Test::Session.new(Rack::MockSession.new(MockPegasus.new, "studio.code.org"))
```

Comment:
Consider using the `build_rack_mock_session` approach Dave pointed out to me [in this PR](https://github.com/code-dot-org/code-dot-org/pull/7911#discussion_r60297776).  It's not well-documented, but basically it provides all the session request methods (`get`, `post`, `last_response`, etc) in the scope of the test.